contracts/auction.sol

```diff
@@ -213,6 +231,7 @@ contract DutchAuction {
         atStage(Stages.AuctionStarted)
     {
         require(msg.value > 0);
+        require(bids[msg.sender] + msg.value < bid_threshold || whitelist[msg.sender]);
```

Comment:
From the comment at the top `Bid value over which` this should this be `<=`.

Reply:
Fixed. Thank you.